platforms/jvm/testing-jvm/src/integTest/groovy/org/gradle/testing/junit/AbstractJUnitIntegrationTest.groovy

```diff
@@ -181,7 +181,7 @@ abstract class AbstractJUnitIntegrationTest extends AbstractTestingMultiVersionI
                 ${testFrameworkDependencies}
             }
             test.${configureTestFramework}
-            test.workingDir = "${testWorkingDir.toURI()}"
+            test.workingDir("${testWorkingDir.toURI()}")
```

Comment:
❓ Isn't our plan to deprecate `void workingDir(Object)`? Then I'd prefer us using non-deprecated syntax here